robot_assignments/ios/jetIOSassignment/testcases/iosAppium.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Resource    ${CURDIR}/../import.robot
+Test Setup      Open Test Application
+Test Teardown       Quit Application
+
+*** Test Cases ***
+Test 01 Verify that login page is shown with information correctly
+    [Tags]  T01
```

Comment:
tags is not correct as instruction ka 
the tags values should be
testid:1001    ios